core/src/main/java/org/openstack4j/api/EndpointTokenProvider.java

```diff
@@ -23,4 +23,12 @@
      * @return the auth token identifier
      */
     String getTokenId();
+    
+    /**
+	 * Gets the micro version.
+	 * If it is seted, the version will be used to set headers.
```

Comment:
```suggestion
	 * If it is setted, the version will be used to set headers.
```

Reply:
Better to say "If it is set to a non-null value, ..." IMO.

Setting it to null results in the same effect as it not being set at all.
The code doesn't care whether it's been set or not; the code adds the new header if this value isn't null.